hospital/hospitalcore/src/org/training/hospital/core/service/impl/DefaultPatientService.java

```diff
@@ -67,41 +63,50 @@ public PatientModel getPatientForUid(String uid) {
 	}
 
 
-public PatientModel releasePatient(String code) {
+	public PatientModel releasePatient(String code) {
 
 
-	Date today = new Date();
-	
+		
+		final Calendar cal= Calendar.getInstance();
+		cal.set(Calendar.HOUR, 0);
+		cal.set(Calendar.MINUTE, 0);
+		cal.set(Calendar.SECOND, 0);
+		cal.set(Calendar.MILLISECOND, 0);
+		Date today = cal.getTime();
+
```

Comment:
possiamo ricavare la data di oggi direttamente facendo new Date() come fatto in altri punti, non ci serve l'oggetto calendar